server/db/migrations/20211005224300_code_project_nonprofit.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('code_project_nonprofit', function (table) {
+    table.integer('code_organization_id').unsigned()
+    table.integer('nonprofit_organization_id').unsigned()
+    table.integer('project_id').unsigned()
+
+    table.timestamp('created_at').defaultTo(knex.fn.now())
+    table.timestamp('updated_at')
+
+    table.foreign('code_organization_id').references('code_organizations.id')
```

Comment:
The linter probably did not complain at you, but would you mind putting foreign and references on their own lines? (and the last one too)

I think that having them all having the same formatting will help with readability.

Reply:
@duaimei ok, I'll fix